jwala-webservices/src/main/java/com/cerner/jwala/ws/rest/v1/service/jvm/JvmServiceRest.java

```diff
@@ -6,34 +6,55 @@
 import com.cerner.jwala.ws.rest.v1.service.jvm.impl.JsonControlJvm;
 import com.cerner.jwala.ws.rest.v1.service.jvm.impl.JsonCreateJvm;
 import com.cerner.jwala.ws.rest.v1.service.jvm.impl.JsonUpdateJvm;
+import io.swagger.annotations.*;
 
 import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
+import java.util.List;
 
+@Api(value = "/jvms", tags = "jvms")
 @Path("/jvms")
 @Produces(MediaType.APPLICATION_JSON)
 public interface JvmServiceRest {
 
     @GET
+    @ApiOperation(value = "Get all the JVMs",
+            response = List.class
+    )
     Response getJvms();
 
     @GET
     @Path("/{jvmId}")
-    Response getJvm(@PathParam("jvmId") final Identifier<Jvm> aJvmId);
+    @ApiOperation(value = "Get a single JVM by ID",
+            response = Jvm.class
+    )
+    Response getJvm(@ApiParam(value = "The JVM's ID", required = true) @PathParam("jvmId") final Identifier<Jvm> aJvmId);
 
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
-    Response createJvm(JsonCreateJvm jsonCreateJvm, @BeanParam AuthenticatedUser aUser);
+    @ApiOperation(value = "Create a JVM",
```

Comment:
Can we add a note? something like, this will creates the jvm configuration within Jwala but only creates the jvm on the server upon generation